.azure-pipelines.yml

```
@@ -155,8 +155,16 @@ jobs:
   - bash: |
       # set options
       set -o nounset errexit pipefail
-      # run tests (exclude pytest.AMReX when running Python tests)
-      ctest --test-dir build --output-on-failure -E AMReX
+      # determine if the build was triggered by a push to the development branch
+      if [[ "$(Build.SourceBranch)" == "refs/heads/development" ]]; then
```

If this does not work, we can also try the following:
```suggestion
      if [[ "$(Build.Reason)" == "IndividualCI" ]]; then
```
See https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables for more documentation.

---

You also need to check the repository name, to be sure (e.g., in case a fork develops on `development` and for some reason has Azure CI on).